### PR TITLE
Rvpyth 61 override login

### PR DESCRIPTION
**Pull request by pika1989**
RVPYTH-61
    To override method for login page

### CODE REVIEW

**Review by yurapa — APPROVED** [review]

**Review by OlhaLesk — APPROVED** [review]

**Review by krizik — APPROVED** [review]

File: restaurant/settings.py [hunk 1]
```diff
@@ -120,6 +120,11 @@
     },
 ]
 
+AUTHENTICATION_BACKENDS = [
+        'django.contrib.auth.backends.AllowAllUsersModelBackend'
```
Comment (ivanna-ostrovets) [inline restaurant/settings.py]: You have inconsistency problem with formatting your code. Please, fix this.

File: restaurant/views/login_view.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+from django.contrib.auth import authenticate, login
+from django.http import HttpResponse, HttpResponseRedirect
+from django.shortcuts import render
+
+
+def login_view(request):
+"""Checks whether user exists in database and has permissions to login.
+"""
```
Comment (ivanna-ostrovets) [inline restaurant/views/login_view.py]: Please, refactor this according to PEP 257.
> The closing quotes are on the same line as the opening quotes. This looks better for one-liners.
[https://www.python.org/dev/peps/pep-0257/#one-line-docstrings](url)

File: restaurant/urls.py [hunk 3]
```diff
@@ -13,15 +13,19 @@
...
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
-    url(r'^account/login/$', auth_views.login, name='login'),
-    url(r'^account/password_reset/$', auth_views.password_reset, name='password_reset'),
-    url(r'^account/password_reset/done/$', auth_views.password_reset_done, name='password_reset_done'),
+    url(r'^index/', index, name='index'),
+    url(r'^account/login/$', login_view, name='login'),
+    url(r'^account/password_reset/$', auth_views.password_reset,
+                                      name='password_reset'),
```
Comment (ivanna-ostrovets) [inline restaurant/urls.py]: In such case you probably shoud align this with an opening bracket.

Something like this:
```python
def function(one, two, three,
             four):
    pass
```

File: restaurant/urls.py [hunk 4]
```diff
@@ -13,15 +13,19 @@
...
+    url(r'^account/password_reset/done/$', auth_views.password_reset_done,
+                                           name='password_reset_done'),
```
Comment (ivanna-ostrovets) [inline restaurant/urls.py]: In such case you probably shoud align this with an opening bracket.

Something like this:
```python
def function(one, two, three,
             four):
    pass
```

**Review by ivanna-ostrovets — CHANGES_REQUESTED** [review]

**Review by yurapa — APPROVED** [review]

**Review by ivanna-ostrovets — APPROVED** [review]

**Pull request merged**